shared/src/business/entities/DocketEntry.unsealEntry.test.ts

```diff
@@ -1,8 +1,22 @@
-import { A_VALID_DOCKET_ENTRY } from './DocketEntry.test';
-import { DOCKET_ENTRY_SEALED_TO_TYPES } from './EntityConstants';
+import { DOCKET_ENTRY_SEALED_TO_TYPES, ROLES } from './EntityConstants';
 import { DocketEntry } from './DocketEntry';
 
 describe('unsealEntry', () => {
+  const mockPrimaryId = '7111b30b-ad38-42c8-9db0-d938cb2cb16b';
+
+  const A_VALID_DOCKET_ENTRY = {
```

Comment:
Is the idea to re-create the test docket entry object as needed in each test rather than to use a centralized test object? I see benefits to both approaches, although I am curious about the decision here given that the object is unchanged in every test. If I just want any old valid docket entry for testing, it is helpful to have one defined rather than re-creating it every time and keeping it up-to-date with any schema changes (even if rare).